.github/workflows/spice_postgres_bench_docker.yml

```diff
@@ -0,0 +1,62 @@
+name: spice-postgres-bench docker
+
+on:
+  push:
+    branches:
+      - qianqian/docker-tpch-postgres
+  pull_request:
+    branches:
+      - qianqian/docker-tpch-postgres
```

Comment:
```suggestion
      - trunk
  pull_request:
    branches:
      - trunk
```

Reply:
Will change to workflow dispatch, this workflow is not supposed to run once we confirm the image contains tpch data we want